contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
I understand the desire but this is unrealistic given our dependencies. Can we have golang 1.7 as a build require? It shouldn't conflict as these packages would only be installed to build.